### PR TITLE
ROC-5625 Add route tests for claim issued and admissions, and fix UI bug

### PR DESCRIPTION
**Pull request by simad**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5265

### Change description

- Adding a first battery of route tests covering a few cases in admissions. This will help to set up a pattern to follow for adding subsequent route tests.
- Also, fixing a UI bug surfaced by the route tests themselves. We were missing a `| numeral` filter in a template, so we were rendering a money amount as a simple integer

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/features/dashboard/routes/index.ts [hunk 1]
```diff
@@ -77,7 +95,317 @@ describe('Dashboard page', () => {
...
+                .expect(res => expect(res).to.be.successful.withText('You’ve admitted part of the claim.'))
+            })
+
+            it('should show pay immediately statuses when the claim is marked as pay immediately and the offer is accepted', async () => {
+              resolveRetrieveByDefendantId('000MC001', '1', acceptedClaim)
+
+              await request(app)
+                .get(Paths.dashboardPage.uri)
+                .set('Cookie', `${cookieName}=ABC`)
+                .expect(res => expect(res).to.be.successful.withText('John Smith accepted your admission of 30'))
```
Comment (andrewwa-kainos) [inline src/test/features/dashboard/routes/index.ts]: Is this exposing a UI bug? Should have the £ prefix, surely?

Comment (simad) [reply src/test/features/dashboard/routes/index.ts]: yes, it is a bug I found and I have a fix ready for it :)

Comment (simad) [reply src/test/features/dashboard/routes/index.ts]: I can actually put the fix in right now, now that I think of it. It was in another PR, but that might take a long(er) time to get merged

Comment (simad) [reply src/test/features/dashboard/routes/index.ts]: Committed the fix :D _scope creep 😆 _

File: src/test/features/dashboard/routes/index.ts [hunk 2]
```diff
@@ -29,7 +47,7 @@ describe('Dashboard page', () => {
 
       it('should return 500 and render error page when cannot retrieve claims', async () => {
         draftStoreServiceMock.resolveFind('claim')
-        claimStoreServiceMock.rejectRetrieveByClaimantId('HTTP error')
+        rejectRetrieveByClaimantId('HTTP error')
```
Comment (KiranVarma02) [inline src/test/features/dashboard/routes/index.ts]: I am guessing the reason for `import * as claimStoreServiceMock from 'test/http-mocks/claim-store'` is to clearly say its a mock, not sure. I think we do this everywhere, should we maybe continue that?

Comment (simad) [reply src/test/features/dashboard/routes/index.ts]: 👍 I think it makes it less legible but probably more explicit, have changed that

**Pull request merged**